Classes/Private/Manager/Video/OCTVideoView.m

```diff
@@ -47,8 +48,16 @@ - (void)setImage:(CIImage *)image
 - (void)drawRect:(CGRect)rect
 {
     if (self.image) {
-        CGFloat scale = self.window.screen.scale;
-        CGRect destRect = CGRectApplyAffineTransform(self.bounds, CGAffineTransformMakeScale(scale, scale));
+
+        glClearColor(0, 0.0, 0.0, 1.0);
+        glClear(GL_COLOR_BUFFER_BIT);
```

Comment:
Yea the pull request as is works. I actually think glClear before every draw is not bad. According to this I think https://developer.apple.com/library/ios/qa/qa1650/_index.html